sensor_msgs/Image camera_image
bool detection_signal